test/e2e/e2eutil.go

```diff
@@ -140,6 +140,22 @@ func waitForStatefulSetCondition(t *testing.T, mdb *mdbv1.MongoDBCommunity, retr
 	})
 }
 
+func WaitForPodReadiness(t *testing.T, isReady bool, containerName string, timeout time.Duration, pod corev1.Pod) error {
```

Comment:
Q: wondering if `timeout` needs to be a parameter, time for the pod to reach ready should be irrespective of the caller? -- maybe we can assume some sane defaults.

Reply:
I agree with you in general though this method has a caveat that it's called from both `PodContainerBecomesReady` and `PodContainerBecomesNotReady` (so both directions: true->false and false->true). So the former passes 3 minutes and the latter - 10 as we have a very long `failureThreshold` for readiness probe. We could try to default each of those two methods instead?

Reply:
> We could try to default each of those two methods instead?

+1